script.extendedinfo/resources/lib/dialogs/DialogVideoList.py

```diff
@@ -48,6 +49,7 @@ class DialogVideoList(DialogBaseList, window_type):
                    "with_networks": addon.LANG(32152),
                    "with_keywords": addon.LANG(32114),
                    "first_air_date": addon.LANG(20416),
+                   "with_runtime": xbmc.getLocalizedString(2050),
```

Comment:
oops, youre right, KodiDevKit uses xbmc.getLocalizedString by default, that's why this ended up here. Will fix for next update.